airflow/sensors/sql.py

```diff
@@ -58,17 +63,27 @@ class SqlSensor(BaseSensorOperator):
     ui_color = '#7c7287'
 
     def __init__(
-        self, *, conn_id, sql, parameters=None, success=None, failure=None, fail_on_empty=False, **kwargs
+        self,
+        *,
+        conn_id,
+        sql,
+        parameters=None,
+        success=None,
+        failure=None,
+        fail_on_empty=False,
+        hook_kwargs=None,
```

Comment:
```suggestion
        hook_kwargs={},
```

And then the line below could be reduced to 

```python
	self.hook_kwargs = hook_kwargs
```

Reply:
Could you take a look https://github.com/apache/airflow/pull/18394 please?